README.md

```diff
@@ -47,9 +45,9 @@ libHttpClient provides a platform abstraction layer for HTTP and WebSocket, and
 
 ## Behavior control
 
-* On UWP, XDK ERA, iOS, and Android, HCHttpCallPerform() will call native platform APIs
+* On GDK, XDK ERA, UWP, iOS, and Android, HCHttpCallPerform() will call native platform APIs
 * Optionally call HCSetHttpCallPerformFunction() to do your own HTTP handling using HCHttpCallRequestGet*(), HCHttpCallResponseSet*(), and HCSettingsGet*()
-* See sample CustomHttpImplWithCurl how to use this callback plus [Curl](https://github.com/curl/curl) to make an HTTP implementation using Curl.
+* See sample CustomHttpImplWithCurl for an example of how to use this callback to make your own HTTP implementation.
 
```

Comment:
maybe the should point to the internal Curl implementation now?

Reply:
Its just saying look at that sample for how to use HCSetHttpCallPerformFunction (not how to use best use Curl)